synapse/lib/stormtypes.py

```diff
@@ -8017,6 +8021,18 @@ async def _methWipeLayer(self):
         view = self.runt.snap.core.getView(viewiden)
         await view.wipeLayer(useriden=useriden)
 
+    async def _methSwapLayer(self):
+
+        iden = self.valu.get('iden')
+        self.runt.reqAdmin(gateiden=iden)
+
+        view = self.runt.snap.core.getView(iden)
+        if not view.parent:
```

Comment:
I think you could still get around this by adding a new view with your target layer and then setting the new view's parent to anything.